go-update/src/main/java/org/reactome/release/goupdate/GoLineProcessor.java

```diff
@@ -162,21 +162,6 @@ static String processLine(String line, String currentGOID, Map<String, Map<Strin
 									addToMultivaluedAttribute(goTerms, currentGOID, line, GoUpdateConstants.RELATIONSHIP_PART_OF_REGEX, GoUpdateConstants.PART_OF);
 									break;
 								}
-								case GoUpdateConstants.REGULATES:
-								{
-									addToMultivaluedAttribute(goTerms, currentGOID, line, GoUpdateConstants.RELATIONSHIP_REGULATES_REGEX, GoUpdateConstants.REGULATES);
-									break;
-								}
-								case GoUpdateConstants.POSITIVELY_REGULATES:
-								{
-									addToMultivaluedAttribute(goTerms, currentGOID, line, GoUpdateConstants.RELATIONSHIP_POSITIVELY_REGULATES_REGEX, GoUpdateConstants.POSITIVELY_REGULATES);
-									break;
-								}
-								case GoUpdateConstants.NEGATIVELY_REGULATES:
-								{
-									addToMultivaluedAttribute(goTerms, currentGOID, line, GoUpdateConstants.RELATIONSHIP_NEGATIVELY_REGULATES_REGEX, GoUpdateConstants.NEGATIVELY_REGULATES);
-									break;
-								}
```

Comment:
Since these keywords are associated with regulation, are these being removed due to data model changes?

Reply:
Those relationships are no longer being used in the Reactome data model.